lb.tf

```diff
@@ -31,12 +31,17 @@ variable "lb_access_logs_expiration_days" {
   default = "3"
 }
 
+variable "lb_internal" {
+  default = false
```

Comment:
Do we want to add this variable to the docs?

Reply:
I'll do this in the absence of Tom.